### PR TITLE
fix: pass CI tests

### PR DESCRIPTION
**Pull request by mallchel**
console error prevents the success of GitHub actions workflow. This PR is just setting up running CI tests, mocking console.error and console.warn

### CODE REVIEW

**Comment by mallchel** [comment]
@aaronabramov 
console error prevents success github actions workflow. This PR is just setting up running CI tests, mocking console.error and console.warn

**Comment by drarmstr** [comment]
cc @aaronabramov

**Comment by mallchel** [comment]
@drarmstr it seems like tests were failing only in this line ```expect(c.textContent).toBe('"ATOMATOMATOM"');```
We can remove mocks for console.error and console.warn for CI

**Comment by drarmstr** [comment]
> @drarmstr it seems like tests were failing only in this line `expect(c.textContent).toBe('"ATOMATOMATOM"');`
> We can remove mocks for console.error and console.warn for CI

If tests are failing there, could it be an actual failure or test we want to fix?  This PR seems dangerous to hide problems?

**Comment by mallchel** [comment]
@drarmstr No, this PR doesn't hide failed tests. If some test fails with the "expect" expression, then the exit code would be non zero and GitHub's actions fail.

Mocking for console methods doesn't make sense at the moment. I can just remove it together with the test-ci script. What do you think?

**Comment by drarmstr** [comment]
I see.  I also fixed the `act()` issue with our local version of #561 (not currently visible on GitHub, the public PR isn't valid).  Anyway, I'll leave this for @aaronabramov to update when he returns next week.

**Comment by mallchel** [comment]
@aaronabramov I'm not sure how to properly test the useRecoilValueLoadable

File: src/recoil_values/__tests__/Recoil_selector-test.js [hunk 1]
```diff
@@ -768,8 +768,8 @@ describe('Async selector resolution notifies all stores that read pending', () =
     expect(c.textContent).toEqual('loading');
 
     act(() => resolve('bar'));
-    await flushPromisesAndTimers();
-    await flushPromisesAndTimers();
+    await act(flushPromisesAndTimers);
```
Comment (drarmstr) [inline src/recoil_values/__tests__/Recoil_selector-test.js:771]: These are no longer necessary with #576

**Review by drarmstr — CHANGES_REQUESTED** [review]

File: src/recoil_values/__tests__/Recoil_selector-test.js (hunk 1 above)
Comment (mallchel) [reply src/recoil_values/__tests__/Recoil_selector-test.js:771]: @drarmstr Yeah. I missed it. fixed

File: src/util/Recoil_mutableSource.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const React = require('React');
+
+function mutableSourceIsExist() {
```
Comment (aaronabramov) [inline src/util/Recoil_mutableSource.js:3]:
```js
function mutableSourceExists() {
```

**Comment by mallchel** [comment]
@aaronabramov updated a PR, moved useMutableSource to Recoil_mutableSource.

**Pull request closed without merging**